pkg/plugins/manager/client/client_test.go

```diff
@@ -50,8 +52,13 @@ func TestQueryData(t *testing.T) {
 				shouldPassThrough: false,
 			},
 			{
-				err:               plugins.ErrPluginGrpcConnectionUnavailableBase.Errorf("unavailable"),
-				expectedError:     plugins.ErrPluginGrpcConnectionUnavailableBase.Errorf("unavailable"),
+				err:               plugins.ErrPluginGrpcConnectionUnavailableBaseFn(context.Background()).Errorf("unavailable"),
+				expectedError:     plugins.ErrPluginGrpcConnectionUnavailableBaseFn(context.Background()).Errorf("unavailable"),
+				shouldPassThrough: true,
+			},
+			{
+				err:               plugins.ErrPluginGrpcConnectionUnavailableBaseFn(context.WithValue(context.Background(), stackIDKey("stackID"), "test-stack")).Errorf("unavailable"),
+				expectedError:     plugins.ErrPluginGrpcConnectionUnavailableBaseFn(context.WithValue(context.Background(), stackIDKey("stackID"), "test-stack")).Errorf("unavailable"),
```

Comment:
I guess these are irrelevant now

Reply:
👍 you are right, the error base is the same no matter the context, so i have removed it.